src/BenchmarksServer/BenchmarksServer.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why stop using the web SDK?

Reply:
Couldn't start the debugger from VS as it's a console app actually